### PR TITLE
[libwebm] Update to 1.0.0.28

### LINKED ISSUE #16406: [libwebm] update port to live-at-HEAD or current HEAD instead of release 1.0.0.27
This issue has been opened to facilitate discussion and to track any work related to updating the libwebm port to the current HEAD of the libwebm upstream master branch.

As @Hoikas mentions [here](https://github.com/microsoft/vcpkg/issues/16398#issuecomment-785663342_), the last tagged release of libwebm (1.0.0.27) is over five years old, and this is the release the portfile uses. There have been many [bug fixes and improvements](https://chromium.googlesource.com/webm/libwebm/+log/refs/heads/master) to libwebm upstream since then.

### PR DESCRIPTION
**Pull request by Cheney-W**
- #### What does your PR fix?  
  1. Fixes https://github.com/microsoft/vcpkg/issues/16406
  2. The tag 1.0.0.28 was generated at 4/24/2021, so I update to this version instead of the current HEAD of the libwebm upstream master branch.
  3. Remove the expired patch files.
  4. Disable the examples, tools, and tests of this port, they can be added as features if needed.
  5. Remove CMAKE_DEBUG_POSTFIX option.

No feature need to be tested.

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

File: ports/libwebm/Fix-cmake.patch [hunk 1]
```diff
@@ -0,0 +1,76 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a83d23b..0f5ede3 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -8,6 +8,8 @@
+ cmake_minimum_required(VERSION 3.2)
+ project(LIBWEBM CXX)
+ 
++set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```
Comment (Hoikas) [inline ports/libwebm/Fix-cmake.patch:9]: Per the maintainer's guide, this should not be done.

File: ports/libwebm/Fix-cmake.patch [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
++)
++
++install(
++  FILES
++    mkvmuxer/mkvmuxer.h
++    mkvmuxer/mkvmuxertypes.h
++    mkvmuxer/mkvmuxerutil.h
++    mkvmuxer/mkvwriter.h
++    mkvparser/mkvparser.h
++    mkvparser/mkvreader.h
```
Comment (Hoikas) [inline ports/libwebm/Fix-cmake.patch:32]: These are the library's internal headers. This change breaks downstream users.

**Review by Hoikas — COMMENTED** [review]
This pull request breaks users of this port. Please either revert or fix it.

File: ports/libwebm/Fix-cmake.patch (hunk 2 above)
Comment (Cheney-W) [reply ports/libwebm/Fix-cmake.patch:32]: Thanks for pointing it out. I will investigate this asap.

Comment (Cheney-W) [reply ports/libwebm/Fix-cmake.patch:32]: @Hoikas 
I tried to fix this issue with PR https://github.com/microsoft/vcpkg/pull/22189, could you please help review it? Thank you!

**Pull request merged**